app/src/main/java/chat/rocket/android/chatrooms/adapter/RoomViewHolder.kt

```diff
@@ -13,8 +13,14 @@ import chat.rocket.common.model.UserStatus
 import kotlinx.android.synthetic.main.item_chat.view.*
 import kotlinx.android.synthetic.main.unread_messages_badge.view.*
 
-class RoomViewHolder(itemView: View, private val listener: (RoomUiModel) -> Unit) :
-    ViewHolder<RoomItemHolder>(itemView) {
+class RoomViewHolder(itemView: View, private val listener: (RoomUiModel) -> Unit, actionsListener: ActionsListener) :
+    ViewHolder<RoomItemHolder>(itemView, actionsListener) {
+
+    init {
```

Comment:
For a better coding style this initializer block comes after the property declaration. 
https://kotlinlang.org/docs/reference/coding-conventions.html